packages/report-server/src/__tests__/reportBuilder/query/QueryBuilder.test.ts

```diff
@@ -220,22 +259,109 @@ describe('QueryBuilder', () => {
                 },
               },
             },
-            query: fullQuery({
+            query: inputQuery({
               startDate: '2020-05-06', // will be ignored - (offset is applied on top of endDate)
               endDate: '2020-10-01',
             }),
           },
-          fullQuery({
+          outputQuery({
             period: `${getPeriodString([2018, 1], [2019, 12])}`,
             startDate: '2018-01-01',
             endDate: '2019-12-31',
           }),
         ],
       ];
 
-      it.each(testData)('%s', (_, { config, query }, expected) => {
+      it.each(testData)('%s', async (_, { config, query }, expected) => {
+        const fullConfig = { transform: [], ...config };
+        const received = await new QueryBuilder(reqContext, fullConfig, query).build();
+        expect(received).toStrictEqual(expected);
+      });
+    });
+  });
+
+  describe('organisation unit params', () => {
+    beforeAll(() => {
+      MockDate.set(CURRENT_DATE_STUB);
+    });
+
+    afterAll(() => {
+      MockDate.reset();
+    });
+
+    describe('organisation unit specs in config', () => {
+      type TestParams = [
+        string,
+        { config: { fetch: Record<string, unknown> }; query: FetchReportQuery },
+        Record<string, unknown>,
+      ];
+
+      const testData: TestParams[] = [
```

Comment:
This is a silly thing because I know it'll work, but I'd like to see a case where none of the org units is a country

Reply:
Sure thing!